help/actions/_posts/2015-02-09-export.md

```diff
@@ -74,27 +74,35 @@ The same format as above but encoded for use as a rich text document (rtf) which
+| symbol                                                                                                                         	| initials             	| different                	| first author: lastname, f. m.other authors: f. m., lastname      	| **%i**    	| , et al.             	|  -Henneken, E. A., G. Muench, L. Holm Nielsen, S. Blanco-Cuaresma, & A. Accomazzi<br>-Kurtz, M. J. & A. Accomazzi                   	|  -Henneken, E. A., G. Muench, et al.<br>-Kurtz, M. J. & A. Accomazzi                         	|
+| text                                                                                                                           	| initials             	| same                     	| lastname, f. m.                                                  	| **%L**    	| , and xxx colleagues 	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., and Accomazzi, A.<br>-Kurtz, M. J. and Accomazzi, A.          	|  -Henneken, E. A., Muench, G., and 4 colleagues<br>-Kurtz, M. J. and Accomazzi, A.           	|
+| symbol                                                                                                                         	| initials             	| same                     	| lastname, f. m.                                                  	| **%l**    	| , et al.             	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., & Accomazzi, A.<br>-Kurtz, M. J., & Accomazzi, A.             	|  -Henneken, E. A., Muench, G., et al.<br>-Kurtz and Accomazzi                                	|
+| none                                                                                                                           	| initials             	| same                     	| lastname, f. m.                                                  	| **%N**    	| , and xxx colleagues 	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., Accomazzi, A.<br>-Kurtz, M. J., Accomazzi, A.                 	|  -Henneken, E. A., Muench, G., and 4 colleagues<br>-Kurtz, M. J., Accomazzi, A.              	|
+| text                                                                                                                           	| initials             	| same                     	| lastname f.m.note no space after first initial                   	| **%e**    	| , and xxx colleagues 	|  -Henneken, E.A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., and Accomazzi, A.<br>-Kurtz, M.J. and Accomazzi, A.            	|  -Henneken, E.A., Muench, G., and 4 colleagues<br>-Kurtz, M.J. and Accomazzi, A.             	|
```

Comment:
Move this row up between `%a` and `%G` to be in alphabetical order with the rest of the examples

Reply:
done.